### DISCUSSION TITLE
Need help on yaml side

### DISCUSSION THREAD

**Discussion by saranraj3195** [community/community#36049] [Actions]
Hey folks!

Have on doubt let me explain my issue,

I have one repo it has 3 yaml files for frontend,backend & admin frontend, If developer push on backend works on every time
my 3 yamls running but frontend & admin frontend folder code have no changes in that time i need only backend yaml only run
remaining two yaml simply do nothing,

For my scenario if user pushes to backend code only backend ci only i need to run

How to configure that!! Kindly help me out from this issue

Thanks for Advance!!!!

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @saranraj3195 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Reply by saranraj3195** [reply]
@queenofcorgis Thanks!! waiting for a positive response

**Comment by qianxi0410** [comment]
see #36046

**Reply by saranraj3195** [reply]
@qianxi0410 Thanks but my job was not triggering the workflow action
